source/wp-content/themes/wporg-gutenberg/gutenberg-content-mobile.php

```diff
@@ -464,12 +464,6 @@
 <div class="wp-block-column is-vertically-aligned-center" style="flex-basis:80%"><!-- wp:paragraph {"fontSize":"level-2"} -->
```

Comment:
Can you update the flex-basis to `100%`?

Reply:
good catch. fixed.